JavaScript/sort/sleepSort.js

```diff
@@ -0,0 +1,24 @@
+/*
+sleepSort -> It is a unique sorting technique in which javascript's  timeout function is used.
+To know what timeout function is , refer to this article : https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout
+
+Drawbacks and constraints of sleepSort:
+1. Sleepsort technique cannot be used in case of negative numbers, because it prints some unwanted result(unsorted array).
+2. It might not be very useful in case of large integers in the array as large integers could cause a long duartion timeout. 
+*/
+
+let numbers = [10,50,3,7,1,75]
```

Comment:
take dynamic input

Reply:
Did you mean taking input in the prompt window? or can you please just help me in taking dynamic input in JS

Reply:
Well u can use Window `prompt` method

Reply:
But using the prompt method, code will become browser-dependent.

Reply:
Yes

Reply:
I think it can be avoided by harding input and documenting it @plazzy99

Reply:
So @SKAUL05 do I have to implement this change or not?

Reply:
@plazzy99 Will review and take decision on this. Meanwhile u resolve the below suggested changes for I/O Messages

Reply:
@plazzy99 please do let me know if I have to implement dynamic input or not.

Reply:
@plazzy99 , this is just a kind reminder, would you still like me to add dynamic input or not, please do answer so that I could commit my code with changes suggested by other mentors as well.

Reply:
ok fine take it hardcoded